internal/services/costmanagement/export_resource_group_resource_test.go

```diff
@@ -171,14 +173,16 @@ resource "azurerm_resource_group_cost_management_export" "test" {
   recurrence_type              = "Monthly"
   recurrence_period_start_date = "%sT00:00:00Z"
   recurrence_period_end_date   = "%sT00:00:00Z"
+  partition_data_enabled               = true
```

Comment:
This would trigger the recreation of the resource so this needs to be removed from the `update` test config.
```suggestion
```